backend/src/main/java/com/htecgroup/skynest/controller/BucketController.java

```diff
@@ -174,4 +169,13 @@ public ResponseEntity<BucketResponse> getBucket(@PathVariable UUID uuid) {
         new ResponseEntity<>(bucketResponse, HttpStatus.OK);
     return bucketResponseEntity;
   }
+
+  @PutMapping("/{uuid}")
+  public ResponseEntity<BucketResponse> editBucket(
+      @Valid @RequestBody BucketEditRequest bucketEditRequest, @PathVariable UUID uuid) {
+    ResponseEntity<BucketResponse> bucketResponseEntity =
+        new ResponseEntity<>(bucketService.editBucket(bucketEditRequest, uuid), HttpStatus.OK);
+    log.info("Bucket is successfully edited");
```

Comment:
should our logs be a bit more detailed? something like:

User <email> (<id>) edited bucket <name> (<id>)

I've already done something like that with [actions](https://github.com/JDP2022SkyNest/SkyNest/pull/114/files#diff-a9284f5ec0b2eca63667d4d3273a734549289b982cca43bac369e4bdc31d3120), so you could use that when it's merged